notebooks/tests/test2_tellseq_C_equal_volume_pooling.py

```diff
@@ -0,0 +1,159 @@
+                "iseq_sequencer": "iSeq",
+                "novaseq_sequencer": "NovaSeqXPlus",
+                "full_plate_fp": "./test_output/QC/Tellseq_plate_df_B.txt",
+                "expt_config_fp": "./test_output/QC/Tellseq_expt_info.yml",
+            }
+    
+            # run notebook with cwd switched to tmp_root so './test_output' points to tmp
+            prev_cwd = os.getcwd()
+            try:
+                os.chdir(tmp_root)
+                pm.execute_notebook(
+                    input_path=os.path.join(self.notebooks_dir, NOTEBOOK),
+                    output_path=str(tmp_root / "executed_test_equal_volume_pooling.ipynb"),
+                    parameters=run_params,
+                    log_output=True,
+                )
+            finally:
+                os.chdir(prev_cwd)
+
+            # collect produced data files under tmp/test_output (skip .ipynb)
+            produced_files = [
+                p for p in tmp_root.glob("test_output/**/*")
+                if p.is_file() and p.suffix.lower() in DATA_EXTS
+            ]
+            self.assertTrue(
+                produced_files,
+                msg="[CHECK] No data outputs (.txt/.csv/.tsv) were produced under tmp/test_output."
+            )
+
+            # --- Expect exactly the names you observed were produced ---
+            expected_names = {
+                "Tellseq_evp_set_col19to24.txt",
+                "Tellseq_plate_df_B.txt",
+                "Tellseq_plate_df_C_set_col19to24.txt",
+                "Tellseq_samplesheet_instrument_iseq_set_col19to24.csv",
+                "Tellseq_samplesheet_spp_iseq_set_col19to24.csv",
+                "Tellseq_samplesheet_spp_novaseqxplus_set_col19to24.csv",
+            }
+
+            # helpful once: see what was actually produced
+            print("Produced files:", sorted(p.name for p in produced_files))
+
+            produced_names = {p.name for p in produced_files}
+            missing = expected_names - produced_names
+            extra   = produced_names - expected_names
+            self.assertFalse(missing, f"Missing outputs: {sorted(missing)}")
+            self.assertFalse(extra,   f"Unexpected outputs: {sorted(extra)}")
+
+            # compare each produced file to its golden
+            for name in sorted(expected_names):
+                produced_fp = next(p for p in produced_files if p.name == name)
+                expected_fp = self._find_expected_path(name)
+
+                # guard: prevent self-compare if paths somehow coincide
+                if Path(expected_fp).resolve() == produced_fp.resolve():
+                    self.fail(
+                        "[GUARD] Produced file path equals expected golden path:\n"
+                        f"  {produced_fp}\n"
+                        "Ensure notebook wrote into tmp and goldens remain in repo."
+                    )
+
+                self._assert_text_files_equal(produced_fp, Path(expected_fp))
```

Comment:
Note that currently the test will ALWAYS fail due to the date situation in the samplesheet files